bcda/alr/gen/synthetic.go

```diff
@@ -95,10 +100,21 @@ var valueGenerator = [...]regexpClosurePair{
 	{regexp.MustCompile("^REV_LINE_CNT$"), func() string { return strconv.Itoa(randomdata.Number(3)) }},
 	{regexp.MustCompile("^B_EM_LINE_CNT_T$"), func() string { return strconv.Itoa(randomdata.Number(3)) }},
 	{regexp.MustCompile("^ASSIGNED_BEFORE$"), func() string { return strconv.Itoa(randomdata.Number(2)) }},
+	{regexp.MustCompile("^COVID19_EPISODE$"), func() string { return strconv.Itoa(randomdata.Number(2)) }},
+	{regexp.MustCompile("^COVID19_MONTH(0[1-9]|1[0-2])$"), func() string { return strconv.Itoa(randomdata.Number(2)) }},
+	{regexp.MustCompile("^(ADMISSION_DT)$"), func() string { return randomDate(minCovidDate, maxCovidDate) }},
+	{regexp.MustCompile("^(DISCHARGE_DT)$"), func() string {
+		return randomEmpty(less,
+			func() string { return randomDate(maxCovidDate, time.Now()) })
+	}},
+	{regexp.MustCompile("^(U071)|(B9729)$"), func() string {
+		return randomEmpty(less,
+			func() string { return randomStringOption([]string{"U071", "B9729"}) })
```

Comment:
I think the value returned is either 0 or 1.

Reply:
Unless Bill said something else about this field.

Reply:
Based on the latest version of his IG, the diagnosis field should look like:
```  "diagnosis" : [
    {
      "condition" : {
        "identifier" : {
          "system" : "http://hl7.org/fhir/sid/icd-10",
          "value" : "U071"
        }
      }
    }
  ],```

BUT, it does need to be adjusted to only be included if the value is 1. Will adjust!